cmd/webhook.go

```diff
@@ -162,25 +155,21 @@ func addVolume(target, added []corev1.Volume, basePath string) (patch []patchOpe
 	return patch
 }
 
-func updateAnnotation(target map[string]string, added map[string]string) (patch []patchOperation) {
-	for key, value := range added {
-		if target == nil || target[key] == "" {
-			target = map[string]string{}
-			patch = append(patch, patchOperation{
-				Op:   "add",
-				Path: "/metadata/annotations",
-				Value: map[string]string{
-					key: value,
-				},
-			})
-		} else {
-			patch = append(patch, patchOperation{
-				Op:    "replace",
-				Path:  "/metadata/annotations/" + key,
-				Value: value,
-			})
-		}
+func updateAnnotation(target map[string]string) (patch []patchOperation) {
+	if target == nil || target[admissionWebhookAnnotationInjectKey] == "" {
+		patch = append(patch, patchOperation{
+			Op:    "add",
+			Path:  "/metadata/annotations/filer-injector-webhook.das-zone.statcan~1inject",
```

Comment:
`filer-injector-webhook.das-zone.statcan~1inject` what does this actually look like on the resource?

Reply:
ok nvm I saw your SC